File: fizzbuzz.py
def main():
  
  for i in list(range(1, 101)):
    s = str(i) + ": "

    if i % 3 != 0 and i % 5 != 0:
      s += str(i)
    else:
      if (i % 3 == 0) and (i % 5 == 0):
        s += "FizzBuzz"
      elif i % 3 == 0:
        s += "Fizz"
      elif i % 5 == 0:
        s += "Buzz"

    print(s)



main()

File: FizzBuzz.java
class FizzBuzz {

    private static void evaluateNumbers(){
        for (int i = 0; i <= 100; i++){
            String result = "";
            if (i % 3 == 0){
                result += "Fizz";
            }
            if (i % 5 == 0){
                result += "Buzz";
            }
            if (i % 3 != 0 && i % 5 != 0){
                result += i;
            }
            System.out.println(result);
        }
    }


    public static void main(String[] args) {
        evaluateNumbers();
    }

}